.buildkite/pipeline.yml

```diff
@@ -56,4 +56,18 @@ steps:
             - "CI_REDSHIFT_DBT_PASS"
             - "CI_REDSHIFT_DBT_USER"
     commands: |
-      bash .buildkite/scripts/run_models.sh redshift
\ No newline at end of file
+      bash .buildkite/scripts/run_models.sh redshift
+
+  - label: ":databricks: Run Tests - Databricks"
```

Comment:
```suggestion
  - label: ":bricks: Run Tests - Databricks"
```

Don't think there's a :databricks: emoji =P

Reply:
i think there is one in buildkite! 
<img width="237" alt="image" src="https://user-images.githubusercontent.com/65564846/216193395-08655c9e-bff9-4dbf-ac52-70d65dc6b8e3.png">